types/camljs/camljs-tests.ts

```diff
@@ -1,4 +1,4 @@
-
+import * as CamlBuilder from 'camljs'
```

Comment:
Generally speaking, a symbol imported with this syntax can't be instantiated with `new`.  Did you mean `import CamlBuilder = require('camljs')`?

Reply:
Wow, thanks for pointing that out! Got me sleepless for half a night exploring the spec, and indeed, this results in a namespace exotic object that doesn't have a constructor.

I was using `import * as xxx from yyy` routinely as a replacement of `var xxx = require(yyy)`.

I guess the right way in TS2.7+ is to use esModuleInterop which addresses this issue specifically (see https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-7.html - section "Support for import d from "cjs" form CommonJS modules with --esModuleInterop")

I'll do the change asap, and thanks again, that was quite an eye opener!